src/content/doc-surrealdb/querying/graphql/surrealist.mdx

```diff
@@ -65,10 +65,15 @@ Additionally, you can start [a serving in Surrealist](/docs/surrealist/concepts/
 
 Next, use the [SurrealQL query editor](/docs/surrealist/concepts/sending-queries) to create some data. For example, you can create a new `user` table with fields for `firstName`, `lastName`, and `email` and add a new user to the database.
 
+In order to allow querying the created table using GraphQL, you will need to explicitly enable graphql using the [`DEFINE CONFIG`](/docs/surrealql/statements/define/config) statement. This will allow you to query the table using GraphQL on a per-database basis.
```

Comment:
```suggestion
In order to allow querying the created table using GraphQL, you will need to explicitly enable GraphQL using the [`DEFINE CONFIG`](/docs/surrealql/statements/define/config) statement. This will allow you to query the table using GraphQL on a per-database basis.
```